test/testdata/lsp/hover.rb

```diff
@@ -102,6 +129,16 @@ def main
   hash = Hash.new("default")
 # ^ hover: T::Hash[T.untyped, T.untyped]
 
+  boo = BigFoo::FOO_CONSTANT
+              # ^^^^^^^^^^^^ hover: Integer(1)
+              # ^^^^^^^^^^^^ hover: The docs for FOO_CONSTANT
+      # ^^^^^^ hover: T.class_of(BigFoo)
+      # ^^^^^^ hover: The docs for BigFoo
+  # TODO .new only works if we definte `initialize`
```

Comment:
This is OK, I think. If a class doesn't have an `initialize`, then there's no point to have docs for `new`.

Reply:
So you can remove the TODO and the nothing assertion, unless you put this here to test a regression for a bug?

Reply:
I'll remove the TODO since the behaviour is now expected, but I think it would be good to keep this for regression

Reply:
7fec53c50

Reply:
lmk what you think -- if the behaviour is rigorous enough we could omit these 2 tests